IDE/VS-AZURE-SPHERE/user_settings.h

```diff
@@ -0,0 +1,66 @@
+#ifndef _USER_SETTINGS_H_
+#define _USER_SETTINGS_H_
+
+#define WOLFSSL_AZSPHERE
+
+/* Client connects to the server with these details. */
+#ifdef CUSTOM_SERVER_CONNECTION
+    #ifndef SERVER_IP
+        #define SERVER_IP "192.168.1.200" /* Local Test Server IP */
```

Comment:
The SERVER_IP is what the client will connect to.  I have the server bind to any interface with 'INADDR_ANY'.  The CUSTOM_SERVER_CONNECTION was made for automated testing of the MT3620 client using a local network server.  The default server details are for www.wolfssl.com so that the user doesn't have to start a server of their own to test the client. I'll get rid of the CUSTOM_SERVER_CONNECTION block of defines because it's a bit excessive and I can do my tests without it.